src/JavaSession1/Bank/Main.java

```diff
@@ -0,0 +1,15 @@
+package JavaSession1.Bank;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Icici icici = new Icici(13f, 1000, 2);
+        Sbi sbi = new Sbi(10f, 200, 1);
+        Boi boi = new Boi(7f, 50000, 10);
```

Comment:
I did not understand what is the point here to define the classes like this. At least the bank should be abstract and other improvements. Let's discuss this during the daily sync meeting.